newIDE/app/src/BehaviorsEditor/BehaviorsEditorService.js

```diff
@@ -16,5 +16,8 @@ export default {
     'Physics2::Physics2Behavior': {
       component: Physics2Editor,
     },
+    'Physics3::Physics3Behavior': {
+      component: Physics2Editor,
```

Comment:
I am a bit skeptical about this component; will the new `Physics3Behavior` use the same component as the Physics2Behaviour or should we revamp it and create a new one even though the b2classes are same? @4ian

Reply:
There is no reason this would stop working. You can try to rename the editor `PhysicsEditor` (remove the 2) to avoid confusion.

Reply:
Oops I thought you misread my question. I am fully aware that it will work; I just wanted to know if it is rational to create a new one instead 🙂

Reply:
Well there is one thing I saw people on discord complain that you can only have one hitbox (unlike polygon based collisions). But if there are modifications to make, why restrain them to Physics 3? I think this is something for another PR.